pubsub/natsstreaming/natsstreaming.go

```diff
@@ -228,7 +228,7 @@ func (n *natsStreamingPubSub) Subscribe(req pubsub.SubscribeRequest, handler fun
 		}
 		pubsub.RetryNotifyRecover(func() error {
 			n.logger.Debugf("Processing NATS Streaming message %s/%d", natsMsg.Subject, natsMsg.Sequence)
-			herr := handler(&msg)
+			herr := handler(context.Background(), &msg)
```

Comment:
Here you can use `n.ctx` instead of `context.Background()`.

Reply:
Done 👍